consensus/checks.go

```diff
@@ -90,7 +90,7 @@ func (consensus *Consensus) onAnnounceSanityChecks(recvMsg *FBFTMessage) bool {
 					"[OnAnnounce] Already in ViewChanging mode, conflicing announce, doing noop",
 				)
 			} else {
-				consensus.startViewChange(consensus.GetCurBlockViewID() + 1)
+				consensus.startViewChange(consensus.GetCurBlockViewID())
```

Comment:
We can remove the input param for startViewChange now. As the view change only depends on last block's viewID and the time difference. In the first 45 seconds after last block is confirmed, it's the current expected leader's job to propose. If it didn't do the job, then view change happens, it's the second 45 seconds and the next leader should be selected. 

So when the viewChange is triggered, based on the time difference and last block, we can already decide the new leaders. No need to refer to consensus.GetCurBlockViewID() anymore.

And with that, we can remove the ViewChangingID actually.

Reply:
I kept it for fallback.  Let me double check.

Reply:
in the getNextViewID function, the viewID is still the fallback mechanism when the chain has not blocks, especially during bootstrap procedure when there is a view change needed.  So, I think we may still need to keep the viewID.  Otherwise, there is no fallback viewID then.